jekyll/_cci2/orb-author.md

```diff
@@ -9,279 +9,289 @@ version:
-* Parameter `steps` are powerful - Wrapping steps provided by the user allows you to encapsulate and sugar things like caching strategies and other more complex tasks, providing a lot of value to users.
+In the background the `orb init` command will be copying and customizing the [Orb Project Template](https://github.com/CircleCI-Public/Orb-Project-Template) based on your inputs. There is a detailed `README.md` file within each directory that contains helpful information specific to that directory.
 
-Refer to [Reusing Config]({{ site.baseurl }}/2.0/reusing-config/) for details and examples of commands, executors and parameters in orbs.
+You will also be asked for the remote git repository URL which we obtained in step 1. Ensure you are using the `HTTPS` URL.
 
-When developing your own orb, you may find it useful to write an inline orb. The section below describes how you can write your own inline orb.
+#### 5) When prompted, in a separate terminal, push the changes up to Github.
+{:.no_toc}
 
-### Writing Inline Orbs
+During the setup process, the `orb init` command will take several steps to prepare your automated orb development pipeline. The modified template code produced by the CLI must be pushed to the repository before the CLI can continue and automatically follow your project on CircleCI.com successfully.
 
-Inline orbs can be handy during development of an orb or as a convenience for name-spacing jobs and commands in lengthy configurations, particularly if you later intend to share the orb with others.
+```bash
+$ git push origin master
+```
 
-To write inline orbs, place the orb elements under that orb's key in the `orbs` declaration in the configuration. For example, if you want to import one orb and then author inline for another, the orb might look like the example shown below:
+For technical reasons, we can not automatically push this code.
+Once complete, return to the CLI window and confirm the changes have been pushed.
 
-{% raw %}
-```yaml
-version: 2.1
-description: # The purpose of this orb
-
-orbs:
-  my-orb:
-    orbs:
-      codecov: circleci/codecov-clojure@0.0.4
-    executors:
-      specialthingsexecutor:
-        docker:
-          - image: circleci/ruby:2.7.0
-    commands:
-      dospecialthings:
-        steps:
-          - run: echo "We will now do special things"
-    jobs:
-      myjob:
-        executor: specialthingsexecutor
-        steps:
-          - dospecialthings
-          - codecov/upload:
-              path: ~/tmp/results.xml
-
-workflows:
-  main:
-    jobs:
-      - my-orb/myjob
+#### 6) The CLI will finish by automatically following the project on CircleCI and generating the first development version fo your orb for testing (a hello-world sample).
+{:.no_toc}
+
+You are now ready to push changes to your orb and automatically publish your orb. We'll go over deploying changes with semver versioning in the [Orb Publishing Process]({{site.baseurl}}/2.0/creating-orbs) docs.
```

Comment:
What do they need to do this? What command?

Reply:
The user will be able to simply push changes now and see the results building on CircleCI. The CLI command will have given them a link to see the building orb.

Publishing the orb live is a little more complex though and so I wanted to steer them to the publishing process page.

Reply:
👍